src/components/NavigationBar.vue

```diff
@@ -22,43 +22,43 @@
 
 <style scoped>
 .navbar {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
```

Comment:
Let's switch all hardcoded pixel references to rem units.

```suggestion
  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.1);
```

The benefit here being that all units will scale equally if the user agent/browser has a custom font-size set for accessibility.

Reply:
I need to get out of the pixel habit!